evalml/automl/automl_algorithm/default_algorithm.py

```diff
@@ -304,7 +328,25 @@ def _create_long_exploration(self, n):
         estimators.sort(key=lambda x: x[1])
         estimators = estimators[:n]
         estimators = [estimator[0].__class__ for estimator in estimators]
-        pipelines = [self._make_split_pipeline(estimator) for estimator in estimators]
+        if is_time_series(self.problem_type):
+            pipelines = [
+                make_pipeline(
+                    X=self.X,
+                    y=self.y,
+                    estimator=estimator,
+                    problem_type=self.problem_type,
+                    sampler_name=self.sampler_name,
+                    parameters=self._pipeline_params,
+                    known_in_advance=self._pipeline_params.get("pipeline", {}).get(
+                        "known_in_advance", None
+                    ),
+                )
+                for estimator in estimators
+            ]
+        else:
+            pipelines = [
+                self._make_split_pipeline(estimator) for estimator in estimators
+            ]
```

Comment:
Noticed that these two chunks of code are the same (for `_create_fast_final` and `_create_long_exploration`) beyond the estimators used. Wondering if it'd be useful to take this chunk and make it a helper function that we can then call and pass in the estimators to.